code.gs

```diff
@@ -536,6 +536,7 @@ Contact.prototype.getLines = function (type, date, format) {
     }
     return typeMatch && event.getProp('day') === date.getDate() && event.getProp('month') === (date.getMonth() + 1);
   }).map(function (event) {
+    var eventLabel = event.getProp('label') || 'OTHER';
```

Comment:
Sorry: I was not really clear before. What I meant was splitting decalration and assignment.

`var line, eventLabel, imgCount;` at line L539

`imgCountevent.getProp('label') || 'OTHER';` at line L564.